storm-core/src/jvm/backtype/storm/utils/Utils.java

```diff
@@ -135,35 +137,68 @@ public static void sleep(long millis) {
     }
 
     public static Map findAndReadConfigFile(String name, boolean mustExist) {
+        InputStream in = null;
+        Boolean confFileEmpty = false;
```

Comment:
There are two variables types in Java: primitive (like boolean, int, long, short and so on), and their boxed counterparts (Boolean, Integer, Long, ...), which are seen as reference types. It's minor, but unless you're working with collections, there is no need to use later one.